src/filter/types.ts

```diff
@@ -5,13 +5,32 @@ export type Mode = "default" | "mobile";
 
 export interface FilterProps {
     children: React.ReactNode | ((mode: Mode) => React.ReactNode);
-    clearButtonDisabled?: boolean | undefined;
+    customLabels?:
+        | Partial<{
+              headerTitle: string | undefined;
+              toggleFilterButtonLabel: string | undefined;
+              doneButtonLabel: string | undefined;
+          }>
```

Comment:
no need to Partial, just include the optional chaining directly on the properties